bin/nvh

```diff
@@ -515,6 +532,18 @@ function is_ok() {
   fi
 }
 
+#
+# Synopsis: can_use_xz
+#
+
+function can_use_xz() {
+  # nvh uses xz on Linux and macOS at this time, if the system supports it
+  if ! ([[ "$(uname -s)" = "Linux" ]] && [[ -n "$(command -v xz)" ]]) \
+  && ! ([[ "$(uname -s)" = "Darwin" ]] && [[ $(echo $(sw_vers -productVersion) | cut -d '.' -f 2) -gt "8" ]]); then
```

Comment:
Is the echo needed?

`$(sw_vers -productVersion | cut -d '.' -f 2)`

Reply:
On second look, no. Updating shortly.